### PR TITLE
Add a new Quote of Paul Graham

### PR DESCRIPTION
**Pull request by ayushsaini121**
We have added new Quotes of Paul Graham.

Author: "Paul Graham"
Quote : "That’s the thing about counter-intuitive ideas, they contradict your intuitions. So, they seem wrong."

### CODE REVIEW

**Comment by nelsonic** [comment]
Source: www.paulgraham.com/before.html

File: quotes.json [hunk 1]
```diff
@@ -6476,5 +6476,9 @@
   {
     "author": "Ziggy",
     "text": "You can complain because roses have thorns, or you can rejoice because thorns have roses."
+  },
+  {
+    "author": "Paul Graham",
+    "text": "That’s the thing about counter-intuitive ideas, they contradict your intuitions. So, they seem wrong."
```
Comment (nelsonic) [inline quotes.json:6482]:
```suggestion
    "text": "That’s the thing about counter-intuitive ideas, they contradict your intuitions. So, they seem wrong.",
    "source": "www.paulgraham.com/before.html",
    "tags": "ideas, intuition, startup"
```

**Review by nelsonic — APPROVED** [review]
Thanks for the addition @ayushsaini121 🎉

**Comment by ayushsaini121** [comment]
Your welcome, and I'm too happy to contribute to your project 😊

**Pull request merged**